test/src/test/java/hudson/util/FormValidationTest.java

```diff
@@ -0,0 +1,33 @@
+package hudson.util;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class FormValidationTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
Can you confirm that the test requires a JenkinsRule?  If this is not included, does test execution fail or generate null pointer exceptions?

Reply:
Yes, the test requires a JenkinsRule because the test checks admin permissions for the Jenkins instance implicitly.